content/articles/building-scalable-projects-using-the-multi-module-architecture-with-spring-boot-and-maven/index.md

```diff
@@ -0,0 +1,115 @@
+# Multi-Module Project Setup in Spring Boot
+
+A three-man organisation in 2019 grew to become a twelve-man team fourteen months later. The growth in team members was necessitated by the growth in the project. However, for them also came an increase in the cost of building, maintaining and scaling the project; they were running the microservices architecture.
+
+In this tutorial, we will be going through the multi-module architecture - which is an drastic improvement on the monolith approach, yet offering as much possibilities of the microservices architecture, while evading the huge demands of microservices. We will be building a model of a hospital management system, taking into account some of the departments that make for a successful medical application.
+
+## Table of contents
+The tutorial will cover:
+- Why is the multi-module approach better than the monolith
+- Advantages of multi-module over microservices architecture
+- Setting a Spring Boot multi-module project
+- How can we make them interact with each other?
+- Gaining the benefits of microservices architecture from the multi-module architecture
+- Bonus
+- Conclusion
```

Comment:
Can we make these hyperlinks?

Reply:
Like you mean they should point to certain URLs?

Reply:
Yes that's right. When you click on these links, it must redirect you to the headings